charts/rpe-send-letter-service/values.aat.template.yaml

```diff
@@ -3,3 +3,5 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   devApplicationInsightsInstrumentKey: "71df5a0f-ebbe-445c-b1d1-d10d349f0970"
+  environment:
+    ENCRYPTION_ENABLED: "false"
```

Comment:
encryption is enabled on AAT ?

Reply:
for staging , in jenkins_cnp encryption is false so it should be false here as well.

Reply:
This is used by the app ? In jenkins file we set for functional tests

Reply:
LetterService is using.

Reply:
in jenkins it is TEST_ENCRYPTION_ENABLED